### PR TITLE
M3-767 Remove duplicative labels

### PR DESCRIPTION
**Pull request by Jskobos**
* Removed titles for textfields where these would duplicate
(or nearly duplicate) the textfield labels.

* Removed in NodeBalancer settings and StackScripts UDFs.

* I checked the remaining areas of the site but may have missed
some, please point me in the right direction if so.

### CODE REVIEW

**Comment by martinmckenna** [comment]
I think there might be some styling work to be done here. These little labels look a bit lonely to me and can probably have a little more emphasis.

Or perhaps we can group together some fields (such as label and password) and give them an appropriate header?

Lonely labels vvvvvv

screen shot 2018-06-22 at 11 43 18 am

**Comment by asauber** [comment]
This branch needs to be brought up to date.

I agree that at least the NodeBalancers settings panels need to be combined into one form. That's one of the reasons this was pointed somewhat high.

**Comment by Jskobos** [comment]
@alioso Andrew suggested (re: Marty's comment above) that you add a prop to the TextField component that will make the label larger, for cases where the field isn't part of a larger section (i.e. on the Linode create from image page, label and password fields).

Those two (label and password) fields also now have extra top margin.

File: src/features/linodes/LinodesCreate/PasswordPanel.tsx [hunk 1]
```diff
@@ -43,13 +43,12 @@ type CombinedProps = Props & WithStyles<ClassNames>;
 
 class PasswordPanel extends React.Component<CombinedProps> {
```
Comment (rjstires) [inline src/features/linodes/LinodesCreate/PasswordPanel.tsx]: This isn't relevant to your PR, and not requesting a change, but this looks weird. I argued that a whole panel for one input was silly a long time ago..

Comment (asauber) [reply src/features/linodes/LinodesCreate/PasswordPanel.tsx]: I specifically requested that this stay separate w.r.t. the design

**Pull request merged**